integration/commands_administration_test.go

```diff
@@ -897,8 +897,9 @@ func TestCommandsAdministrationDBStats(t *testing.T) {
 	assert.InDelta(t, 37_500, doc.Remove("storageSize"), 37_450)
 	assert.InDelta(t, 49_152, doc.Remove("totalSize"), 49_100)
 
-	// TODO assert.Empty(t, doc.Keys())
-	// https://github.com/FerretDB/FerretDB/issues/727
```

Comment:
Do we want to track `fsUsedSize` and `fsTotalSize` in this issue? Those are flags not implemented.